ci/playbooks/deploy_watcher_service.yaml

```diff
@@ -8,6 +8,15 @@
     KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
     PATH: "{{ cifmw_path }}"
   tasks:
+    - name: Patch telemetry service
```

Comment:
I think we should check if enableRemoteWriteReceiver is previously enabled (as in the non-unijobs) to avoid that patch if not needed.

We should wait for openstackcontrolplane ready condition, not only for the prometheus pod pod.

Reply:
Done!